commit c2659997ef88171c278c25c09e89057aea848a7b
Author: Ondřej Čertík <ondrej@certik.us>
Date:   Tue May 6 13:25:29 2025 -0600

    Workaround for namelist support

diff --git a/driver.f90 b/driver.f90
index a5fe28a..1230319 100644
--- a/driver.f90
+++ b/driver.f90
@@ -17,11 +17,13 @@ character(:), allocatable, intent(out) :: input_txt
 integer, intent(out) :: n_tokens_to_generate
 character(1024) :: input_txt2
 integer :: u, ios
-namelist / input_fastGPT / n_tokens_to_generate
 allocate(character(0) :: input_txt)
 input_txt = ""
 open(newunit=u, file=filename, status="old")
-read(u, input_fastGPT)
+n_tokens_to_generate = 20
+read(u, "(a)") input_txt2
+read(u, "(a)") input_txt2
+read(u, "(a)") input_txt2
 do
     read(u, "(a)", iostat=ios) input_txt2
     if (ios /= 0) exit
